<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>525</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Visitors Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>851</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(101, 151, 0)</string>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(140, 0, 0)</string>
    </property>
    <property name="text">
     <string>Удаление записи</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="abNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Введите желаемый номер абонемента</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер абонемента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Введите ФИО клиента</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер телефона:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип абонемента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Осталось посещений:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Введите номер телефона клиента</string>
    </property>
   </widget>
   <widget class="QComboBox" name="abType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выберете тип абонемента</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="remVis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>63</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Введите число занятий, оплаченое клиентом</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Введите номер абонемента для удаления</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер абонемента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>651</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Введите номер абонемента для поиска</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(101, 151, 0)</string>
    </property>
    <property name="text">
     <string>Засчитать посещение</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="toolTip">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
